packages/trace-viewer/src/ui/networkTab.tsx

```diff
@@ -371,3 +379,120 @@ function filterEntry({ searchValue, resourceType }: FilterState) {
     return typePredicate(entry.contentType) && entry.name.url.toLowerCase().includes(searchValue.toLowerCase());
   };
 }
+
+// The following function is derived from Chromium's source code
```

Comment:
Put this into a separate `third_party/curl.ts` file with a proper copyright instead.